# Sun 2005-01-15 - Jason Watson
# while trying to build on RedHat Enterprise 3 with automake-1.6, I ran into this error from ./configure
# if --without-doxygen is used, the DOXYGEN_TRUE and DOXYGEN_FALSE vars are never defined
# the following patch resolved the issue by declaring the DOXYGEN_TRUE and DOXYGEN_FALSE vars first for all cases
#
# uname -a = Linux localhost.localdomain 2.4.21-15.EL #1 Thu Apr 22 00:27:41 EDT 2004 i686 i686 i386 GNU/Linux
# automake --verson = automake (GNU automake) 1.6.3
# autoconf --version = autoconf (GNU Autoconf) 2.57
###################################################################
#
# configure: doxygen disabled by configuration (use --with-doxygen to enable).
# configure: error: conditional "DOXYGEN" was never defined.
# Usually this means the macro was only invoked conditionally.
#
###################################################################
diff -urN libhid-0.2.10.orig/m4/md_check_doxygen.m4 libhid-0.2.10.new/m4/md_check_doxygen.m4
--- libhid-0.2.10.orig/m4/md_check_doxygen.m4    2005-01-15 17:59:22.000000000 -0500
+++ libhid-0.2.10.new/m4/md_check_doxygen.m4    2005-01-15 17:58:59.000000000 -0500
@@ -1,5 +1,6 @@
AC_DEFUN([MD_CHECK_DOXYGEN],
  [
+    AM_CONDITIONAL([DOXYGEN], [false])
    MD_CHECK_ARG_WITH(doxygen, doxygen,
      [
        AC_CHECK_PROG(HAVE_doxygen, $path, yes)
